cf_xarray/units.py

```diff
@@ -15,9 +15,11 @@
 )
 
 # Create registry, with preprocessors for UDUNITS-style powers (m2 s-2) and percent signs
+# also change salinity unit 1e-3 to psu
 units = pint.UnitRegistry(
     autoconvert_offset_to_baseunit=True,
     preprocessors=[
+        lambda string: string.replace("1e-3", "psu"),
```

Comment:
i guess this is the best approach. unless @jthielen has ideas.

Reply:
Do you like psu vs. changing to "1" as the unit?

Also I am able to use the same registry with `pint-pandas` so that is really handy! And import orders matters completely. I had inadvertently done the import order wrong because I had dependencies within something I imported first that messed up the order.

Reply:
I think I prefer `"1"`.

> Also I am able to use the same registry with pint-pandas so that is really handy! 

nice!

> And import orders matters completely. 

Looks like pint-xarray should add a little FAQ entry on checking this. cc @keewis

Reply:
Ok salinity units of 1e-3 are now changed to 1.

I'm happy with this when others are!

Reply:
> Looks like pint-xarray should add a little FAQ entry on checking this.

will do. I'm still hoping to eventually figure out a way to completely avoid this, though.

Reply:
that would be better!